### PR TITLE
OptiPlex_7010_9010: Add v0.1.0 release

### PR DESCRIPTION
**Pull request by philipandag**

### CODE REVIEW

File: boards/Dell/OptiPlex_7010_9010/results.csv [hunk 1]
```diff
@@ -0,0 +1,197 @@
+Test name,Test ID,OptiPlex 7010/9010  v0.1.0-rc1
```
Comment (BeataZdunczyk) [inline boards/Dell/OptiPlex_7010_9010/results.csv:1]: Please change the name in the table and file name from `OptiPlex 7010/9010  v0.1.0-rc1` to `OptiPlex 7010 v0.1.0`. We did not test the 9010 model.  The folder name is fine `boards/Dell/OptiPlex_7010_9010`, so let's leave it as is.

File: boards/Dell/OptiPlex_7010_9010/results.csv [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
```
Comment (BeataZdunczyk) [inline boards/Dell/OptiPlex_7010_9010/results.csv:197]: Remove unnecessary empty cells

**Review by BeataZdunczyk — CHANGES_REQUESTED** [review]

File: boards/Dell/OptiPlex_7010_9010/results.csv (hunk 2 above)
Comment (philipandag) [reply boards/Dell/OptiPlex_7010_9010/results.csv]: That's an error in the `update_results.py` script, there is a PR fixing it here https://github.com/Dasharo/osfv-results/pull/9

https://github.com/Dasharo/osfv-results/compare/bafe39bbc5a79ee19b50f689fe69fea5002f891b..c79b8fc895dd83df865ee79ce902c7f9e7921281#diff-3bb95ffd5bea876e4f909e367c3747236e1d3b97ecca8d2ebbc64e90830f9551L162-L197

File: boards/Dell/OptiPlex_7010_9010/results.csv (hunk 1 above)
Comment (philipandag) [reply boards/Dell/OptiPlex_7010_9010/results.csv]: https://github.com/Dasharo/osfv-results/compare/c79b8fc895dd83df865ee79ce902c7f9e7921281..346e74c0c7b268eea9e7d9d2922ffe8e5a2f3e31#diff-d81e7cc298c4f5085a8bf46384ea55a4109c76a70a9cb924414085e9512aff0cL1

Changed the column name to `Optiplex 7010` and the filename to `v0.1.0-results.csv` instead so it looks like the VP66xx results

File: README.md [hunk 3]
```diff
@@ -13,6 +13,8 @@ mainboards:
 - [MSI Z790-P PRO](./boards/msi/ms7e06) -- project [Raptor Lake Desktop](https://nlnet.nl/project/RaptorLake/)
 - [NovaCustom V54](./boards/NovaCustom/V54x_MTL/)
 - [NovaCustom V56](./boards/NovaCustom/V56x_MTL/)
+- [Protectli VP66xx](./boards/Protectli/VP66xx/)
```
Comment (BeataZdunczyk) [inline README.md:16]: The results for v66xx have already been published and should not be included here.

File: boards/Protectli/VP66xx/v0.9.0-results.csv [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+"Check core count (HT Enabled, P: All, E: All) (Ubuntu)",CCC004.001,PASS,PASS
+"Check core count (HT Disabled, P: All, E: 0) (Ubuntu)",CCC005.001,PASS,PASS
+"Check core count (HT Disabled, P: All, E: 0) (Ubuntu)",CCC006.001,PASS,PASS
+"Check core count (HT Enabled, P: 1, E: A) (Ubuntu)",CCC007.001,PASS,PASS
+"Check core count (HT Disabled, P: 1, E: A) (Ubuntu)",CCC008.001,PASS,PASS
+"Check core count (HT Enabled, P: 1, E: 1) (Ubuntu)",CCC009.001,PASS,PASS
+"Check core count (HT Disabled, P: 1, E: 1) (Ubuntu)",CCC010.001,PASS,PASS
+"Check core count (HT Enabled, P: A, E: 1) (Ubuntu)",CCC011.001,PASS,PASS
+"Check core count (HT Disabled, P: A, E: 1) (Ubuntu)",CCC012.001,PASS,PASS
+
```
Comment (BeataZdunczyk) [inline boards/Protectli/VP66xx/v0.9.0-results.csv:90]: The results for v66xx have already been published and should not be included here.

File: boards/Protectli/VP66xx/results.csv.license [hunk 5]
```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: 2024 3mdeb Sp. z o. o.
+
+SPDX-License-Identifier: CC-BY-SA-4.0
```
Comment (BeataZdunczyk) [inline boards/Protectli/VP66xx/results.csv.license:3]: The results for v66xx have already been published and should not be included here.

**Review by BeataZdunczyk — CHANGES_REQUESTED** [review]

File: boards/Protectli/VP66xx/results.csv.license (hunk 5 above)
Comment (philipandag) [reply boards/Protectli/VP66xx/results.csv.license:3]: Yes, this branch needs to be rebased after publishing VP66xx results.

**Review by BeataZdunczyk — APPROVED** [review]

**Pull request merged**